src/main/java/mate/jdbc/dao/DriverDaoImpl.java

```diff
@@ -74,7 +74,7 @@ public Driver create(Driver driver) {
     @Override
     public Driver update(Driver driver) {
         String query = "UPDATE drivers "
-                + "SET name = ?, license_number = ? "
+                + "SET name = ?, license = ? "
```

Comment:
no need to change this. class field is `licenseNumber`, two words, so column should also be two words